.github/workflows/grid_client_tests.yml

```diff
@@ -2,32 +2,34 @@ name: Grid Client Tests
 
 on:
   schedule:
-    - cron: "0 0 * * *"
+    - cron: 0 0 * * *
   workflow_dispatch:
     inputs:
       network:
         type: choice
-        description: 'which network to run tests on'
+        description: "Which network to run tests on"
         required: true
-        default: 'dev'
+        default: "dev"
         options: 
-        - 'dev'
-        - 'qa'
-        - 'test'
-        - 'main'
-
+        - "dev"
+        - "qa"
+        - "test"
+        - "main"
 jobs:
   tests:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     env:
       NETWORK: ${{ github.event.inputs.network }}
       RMB_PROXY: true
       STORE_SECRET: secret
       MNEMONIC: ${{ secrets.MNEMONIC }}
     steps:
-      - uses: actions/checkout@v2
+      - name: set network for Schedule
+        if: ${{env.NETWORK}} == ""
+        run: echo "NETWORK=dev" >> $GITHUB_ENV
+      - uses: actions/checkout@v4
       - name: Set up node 18
```

Comment:
Maybe change this to Set up node version?

Reply:
Changed in [195c3d3](https://github.com/threefoldtech/tfgrid-sdk-ts/pull/1454/commits/195c3d369550071f470b1d4e66a524b74af13cc0)